paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -73,7 +73,7 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
             configuration = paymentSheetConfiguration,
         )
 
-        reportLoadResult(elementsSessionResult)
+        reportLoadResult(elementsSessionResult, isGooglePaySupported())
```

Comment:
Why do we need to include it here? Can’t we just start adding `googlePaySupported` from `onLoadSucceeded()` going forward?

Reply:
Doesn't necessarily need to be included here, can just make `reportLoadResult` suspendable.